README.md

```diff
@@ -531,3 +531,43 @@ To run an end-to-end test against rinkeby
 - Similar to Rinkeby section (replace RINKEBY with GOERLI) and use `--network goerli`.
 - Faucets: [Simple](https://goerli-faucet.slock.it),
   [Social](https://faucet.goerli.mudit.blog/).
+
+### Running the smoke tests on some deployed contract
+
+#### Deploy the CAPE contract on Goerli
+
+If the CAPE contract is already deployed you can skip to the next section.
+
+- Generate the faucet manager keys, e.g.:
+
+```
+> [nix-shell:~/repositories/Translucence/cape]$ cargo run --bin faucet-wallet-test-setup
+    Finished dev [unoptimized + debuginfo] target(s) in 0.19s
+     Running `target/debug/faucet-wallet-test-setup`
+Faucet manager encryption key: USERPUBKEY~ekZ5WsvbY5c74x9ZtW6_mwyxsCo9UMNreIecAf-4Vq0gAAAAAAAAAPGTZ7plvedP88dKH2J7i7z3fD3CKfSK9iC1mxTOdH4F1w
+Faucet manager address: ADDR~ekZ5WsvbY5c74x9ZtW6_mwyxsCo9UMNreIecAf-4Vq1L
+CAPE_FAUCET_MANAGER_MNEMONIC="hello hello hello hello hello hello hello hello hello hello hello good"
+CAPE_FAUCET_MANAGER_ENC_KEY=0xf19367ba65bde74ff3c74a1f627b8bbcf77c3dc229f48af620b59b14ce747e05
+CAPE_FAUCET_MANAGER_ADDRESS_X=0x2d56b8ff019c87786bc3503d2ab0b10c9bbf6eb5591fe33b9763dbcb5a79467a
+CAPE_FAUCET_MANAGER_ADDRESS_Y=0x21682e859e3d9eedb883221a4d5aab1157a154196a1ad93df975b43ba3fd4f4e
+```
+
+- Update the deployment script in order to handle this new key. Follow the instructions at
+  `contracts/deploy/00_cape.ts`, line 34
+
+- Deploy the CAPE contract on Goerli
+
+  `hardhat deploy --network goerli`
+
+#### Run the tests
+
+Set the following environment variables in the .env file:
+
+- `CAPE_FAUCET_MANAGER_MNEMONIC`: is the mnemonic for the Faucet manager's keys.
+- `DEPLOYED_CAPE_CONTRACT_ADDRESS`: is the address of the CAPE contract to be tested (obtained from running the previous hardhat command)
```

Comment:
Can we use `CAPE_CONTRACT_ADDRESS` which is the env var used for this elsewhere in the repo?

Reply:
This is part of #899